charts/helm-controller-operator/values.yaml

```
@@ -0,0 +1,175 @@
+# Default values for helm-controller-manager
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# -- ****** Value for the helm-controller *******
+helmController:
+  manager:
+    # -- Number of desired pods.
+    replicaCount: 1
+    # -- Pods annotation.
+    podannotations: {}
+      # prometheus.io/port: "8080"
+      # prometheus.io/scrape: "true"
+    terminationGracePeriodSeconds: 600
+    progressDeadlineSeconds: 600
+    revisionHistoryLimit: 10
+    podAnnotations: {}
+    extraArgs: {}
+    # log-level: info
+    image:
+      repository: ghcr.io/fluxcd/helm-controller
+      tag: v0.23.0
+      pullPolicy: IfNotPresent
+    livenessProbe:
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      httpGet:
+        path: /healthz
+        port: healthz
+        scheme: HTTP
+    readinessProbe:
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      httpGet:
+        path: /readyz
+        port: healthz
+        scheme: HTTP
+    ressources:
+      limits:
+        cpu: "1"
+        memory: 1Gi
+      requests:
+        cpu: 100m
+        memory: 64Mi
```

No resources by default (and resources takes one s)
```suggestion
    resources: {}
    #  limits:
    #    cpu: "1"
    #    memory: 1Gi
    #  requests:
    #    cpu: 100m
    #    memory: 64Mi
```